docs/level_0/basic_concepts.md

```diff
@@ -0,0 +1,65 @@
+# Basic Concepts
+
+This document describes the basic concepts, artifacts and their relationships of Landscaper. 
+
+## DeployItem
+
+The main objective of Landscaper is to set up large cloud environments with complex relationships consisting of many
+small installation tasks like helm deployments, network setup etc. These elementary installation tasks are defined by 
+DeployItems. There are different types of DeployItems for example for deploying helm charts, plain kubernetes manifests 
+or terraform configurations.
+
+A DeployItem can be configurable such that it can be used in different scenarios. Typical examples of configurable parts
+of a DeployItem are the target cluster or namespace where a helm chart should be deployed to. A DeployItem can also 
+define output parameter for data it creates and which can be consumed by others.
+
+## Blueprint
+
+Blueprints are reusable collections of combined installation tasks which could be parameterized through import
+parameters. They describe system setups consisting of a set of installations task and their dependencies.
```

Comment:
```suggestion
Blueprints are reusable collections of combined installation tasks which could be parameterized through import
parameters.
```
the same information is already said as `reusable collections of combined installation tasks`. 
alternativly we could have something like:
`Blueprints are reusable system setups which could be parameterized through import
parameters. System setups are collections of combined installations task and their dependencies.`